package.json

```diff
@@ -29,6 +29,7 @@
     "detect-browser": "^4.8.0",
     "i18n": "^0.8.6",
     "jdenticon": "^2.2.0",
+    "jitsi-iframe-api": "^1.0.0",
```

Comment:
It would be nice to publish it in `@aeternity` team on npm (I can't add you to that team, but you can ask me to publish it) and to make this package point to some git repo with sources.

Reply:
Yes, Milen just asked me to do it in that way.